src/main/java/io/qameta/allure/maven/AllureGenerateMojo.java

```diff
@@ -75,22 +75,10 @@
     private String reportDirectory;
 
     /**
-     * Serve timeout parameter in seconds.
+     * Report timeout parameter in seconds.
      */
-    @Parameter(property = "allure.serve.timeout")
-    protected String serveTimeout;
-
-    /**
-     * Serve host parameter.
-     */
-    @Parameter(property = "allure.serve.host")
-    protected String serveHost;
-
-    /**
-     * Serve port parameter.
-     */
-    @Parameter(property = "allure.serve.port", defaultValue = "0")
-    protected Integer servePort;
+    @Parameter(property = "allure.report.timeout", defaultValue = "60")
```

Comment:
```suggestion
    @Parameter(property = "allure.report.timeout", defaultValue = "600")
```
Can I suggest a longer default value? 1 minute is not enough for big reports on Windows machines.

Reply:
I know it's to short, that's why this pull was created. This value will be configurable from now, so you would be able to set it to any value you like. 60sec worked here for a long time and from all allure-maven users just two of us starts to complains about that. @baev which default value would be sufficient in your opinion?

Reply:
I'm ok with one minute timeout by default